components/link/link-styles.js

```diff
@@ -0,0 +1,51 @@
+import '../colors/colors.js';
+import { css, unsafeCSS } from 'lit';
+import { _isValidCssSelector } from '../typography/styles.js';
```

Comment:
Do we want to move this somewhere more general?

Reply:
Do we want to remove the `_` then? Or we could put it in `internal`?

Reply:
🤷‍♂️ . I think the `_` was used for other helpers to make it super duper clear that it's not available for public use.

Reply:
I'm good either way. Like ideally one day we could migrate core to use the `exports` stuff in `package.json`. That way we could actually prevent people from importing things we want kept internal.

Reply:
I did both lol, super easy to move around and there's only going to be a couple of uses if we want to rename